### PR TITLE
fix: build with Xcode 12.4 SQPIT-781

### PR DESCRIPTION
**Pull request by billypchan**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/SQPIT-781" title="SQPIT-781" target="_blank">TaskSQPIT-781</a>  Build frameworks compiles on both Xcode 12 and 13
  </td></table>
  <br />
 

# What's new in this PR?

### Issues

When building `wire-ios` with Xcode 12.4, got the error: 
```
Failed to build module 'WireProtos' from its module interface; the compiler that produced it, 'Apple Swift version 5.5.1 (swiftlang-1300.0.31.4 clang-1300.0.29.6)', may have used features that aren't supported by this compiler, 'Apple Swift version 5.3.2 (swiftlang-1200.0.45 clang-1200.0.32.28)'
```

### Causes 

this framework is build with Xcode 13 and not supported by Xcode 12.4

### Solutions

Build with Xcode 12.4

### dependency 
- [ ] https://github.com/wireapp/wire-ios-shared-resources/pull/70

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -3,7 +3,7 @@ resources:
   - repository: wire-ios-shared-resources
     type: github
     name: wireapp/wire-ios-shared-resources
-    ref: refs/heads/master # Branch to fetch the jobs template from
+    ref: refs/heads/billypchan-patch-2 # Branch to fetch the jobs template from
```
Comment (johnxnguyen) [inline azure-pipelines.yml:6]: Is this final? Or are you planning to give a more descriptive name?

**Review by KaterinaWire — APPROVED** [review]

File: azure-pipelines.yml (hunk 1 above)
Comment (billypchan) [reply azure-pipelines.yml:6]: done

Comment (billypchan) [inline azure-pipelines.yml:6]:
```suggestion
    ref: refs/heads/master # Branch to fetch the jobs template from
```

**Review by johnxnguyen — APPROVED** [review]

**Comment by billypchan** [comment]
release patch

**Comment by zenkins** [comment]
Hello, @billypchan! Merging PR for command **release patch** for PR #**60**. Will create a **patch** release after merged.
Next version is 30.1.1.

**Pull request merged**